desktop/app/lib/config/index.js

```diff
@@ -1,5 +1,5 @@
-const pkg = require( '../../../../desktop/package.json' );
-const config = require( '../../config.json' );
+const pkg = require( '../../../package.json' );
```

Comment:
@sandymcfadden I ended up catching a bug in the code - looks like these import paths were wrong (might have slipped through the cracks when we ported things over during the rewrite). 

It turns out we can use the generated config to override behaviors at run time. In fact, there already exists a check for whether an `updater` variable is set on the config. I override here in this file, based on the presence of a `WINDOWS_STORE` environment variable we can set at build time.